apps/scan/backend/src/electrical_testing/app.ts

```diff
@@ -104,7 +115,14 @@ function buildApi({
         usbDriveStatus.status === 'mounted'
           ? join(usbDriveStatus.mountPoint, 'ballot-images')
           : workspace.ballotImagesPath;
-      const allFileNames = await readdir(basedir);
+      const allFileNames = await readdir(basedir).catch((error): string[] => {
```

Comment:
Unrelated change, but this was failing for me out-the-box, since I didn't have the right directory structure in my mock USB drive